app/adapters/cats.js

```diff
@@ -74,6 +75,12 @@ CATS.Adapter.Cats = Classify({
     add_user: function(v) {
         var user = new CATS.Model.User();
         user.id = v['account_id'];
+        if (typeof v["time_diff_minutes"] !== "undefined"){
+			user.time_offset = v["time_diff_minutes"];
```

Comment:
1) Indent with spaces, not tabs
2) Maybe just `user.time_offset = v["time_diff_minutes"] || 0` ?